vnext/package.json

```diff
@@ -8,7 +8,7 @@
     "directory": "vnext"
   },
   "description": "ReactNative Windows implementation using react-native's c++ ReactCommon bridge",
-  "main": "Libraries/react-native/react-native-implementation.windows.js",
+  "main": "react-native-windoww-implementation.js",
   "typings": "Libraries/react-native/typings-index.d.ts",
```

Comment:
>typings": "Libraries/react-native/typings-index.d.ts", [](start = 3, length = 54)

TODO:  Not sure what needs to be done with this guy.

Reply:
We'll need to entry files.  One for react-native-windows, and one for react-native.  The main and typings entry in package.json should point to the entry file that provides react-native-windows's implementation.

Reply:
I refactored this into a separate ts file just for the react-native-windows types (eg Popup and friends).  afaict we never supplied any typings for react-native itself.

---
In reply to: [405076291](https://github.com/microsoft/react-native-windows/pull/4522#discussion_r405076291) [](ancestors = 405076291)